### ISSUE TITLE
use of Object.defineProperty breaks ie8 again

### ISSUE THREAD

**Issue by xcatliu** [rackt/react-redux#227]
At https://github.com/rackt/react-redux/issues/133, we fixed the issue within `v3.1.0`.

But it appears again in `v4.0.3`.

Here is the file `lib/index.js` in `v3.1.0` and `v4.0.3`.

`v3.1.0`
```js
'use strict';

exports.__esModule = true;

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _componentsCreateAll = require('./components/createAll');

var _componentsCreateAll2 = _interopRequireDefault(_componentsCreateAll);

var _createAll = _componentsCreateAll2['default'](_react2['default']);

var Provider = _createAll.Provider;
var connect = _createAll.connect;
exports.Provider = Provider;
exports.connect = connect;
```

`v4.0.3`
```js
'use strict';

exports.__esModule = true;

var _Provider = require('./components/Provider');

Object.defineProperty(exports, 'Provider', {
  enumerable: true,
  get: function get() {
    return _Provider.default;
  }
});

var _connect = require('./components/connect');

Object.defineProperty(exports, 'connect', {
  enumerable: true,
  get: function get() {
    return _connect.default;
  }
});
```

**Comment by xcatliu** [comment]
It seems to be a problem with babel@6.

What `.babelrc` should we set to avoid the use of `defineProperty`?

**Comment by gaearon** [comment]
Sorry for screwing this up. Should be fixed in 4.0.4.

**Comment by xcatliu** [comment]
Thanks! Works fine with [v4.0.4](https://github.com/rackt/react-redux/releases/tag/v4.0.4).
